src/index/TextIndexBuilder.cpp

```diff
@@ -161,7 +161,7 @@ cppcoro::generator<WordsFileLine> TextIndexBuilder::wordsInTextRecords(
       if (!isLiteral(text)) {
         continue;
       }
-      WordsFileLine entityLine{text, true, contextId, 1, true};
+      WordsFileLine entityLine{std::string{text}, true, contextId, 1, true};
```

Comment:
Same here, please add a comment explaining why the explicit constructor is necessary.